src/docfx/config/Config.cs

```diff
@@ -60,9 +60,9 @@ internal class Config : PreloadConfig
         public Dictionary<string, GroupConfig> Groups { get; } = new Dictionary<string, GroupConfig>();
 
         /// <summary>
-        /// Gets whether to output JSON model.
+        /// Gets whether to generate static HTML output
         /// </summary>
-        public bool OutputJson { get; private set; } = false;
```

Comment:
>OutputJson [](start = 20, length = 10)

this also seems like a breaking change to docs-pipeline - please make sure you have plans for corresponding updates

Reply:
docs-pipeline haven't used those two configs for now, and the default value is still the same with before, now extra work.

Reply:
docs-pipeline haven't used those two configs for now, and the default value is still the same with before, now extra work.

---
In reply to: [411904171](https://github.com/dotnet/docfx/pull/5806#discussion_r411904171) [](ancestors = 411904171)